extension/src/popup/views/AccountCreator/index.tsx

```diff
@@ -67,106 +72,127 @@ export const AccountCreator = () => {
+        data-testid="account-creator-view"
+        hasNoTopPadding
+      >
         <Formik
           initialValues={initialValues}
           onSubmit={handleSubmit}
           validationSchema={AccountCreatorSchema}
         >
           {({ isValid, dirty, isSubmitting, errors, touched }) => (
-            <Onboarding layout="half">
-              <OnboardingHeader>{t("Create a password")}</OnboardingHeader>
-              <Form>
-                <OnboardingOneCol>
-                  <FormRows>
-                    <Field name="password">
-                      {({ field }: FieldProps) => (
-                        <Input
-                          data-testid="account-creator-password-input"
-                          fieldSize="md"
-                          autoComplete="off"
-                          id="new-password-input"
-                          placeholder={t("New password")}
-                          type="password"
-                          error={
-                            authError ||
-                            (errors.password && touched.password
-                              ? errors.password
-                              : "")
-                          }
-                          {...field}
-                        />
-                      )}
-                    </Field>
-                    <Field name="confirmPassword">
-                      {({ field }: FieldProps) => (
-                        <Input
-                          data-testid="account-creator-confirm-password-input"
-                          fieldSize="md"
-                          autoComplete="off"
-                          id="confirm-password-input"
-                          placeholder={t("Confirm password")}
-                          type="password"
-                          error={
-                            authError ||
-                            (errors.confirmPassword && touched.confirmPassword
-                              ? errors.confirmPassword
-                              : null)
-                          }
-                          {...field}
-                        />
-                      )}
-                    </Field>
-                  </FormRows>
-
-                  <PasswordRequirements />
+            <div className="AccountCreator">
+              <Onboarding layout="half">
+                <Card variant="secondary">
+                  <Heading as="h2" size="xs" weight="semi-bold">
+                    {t("Create a Password")}
+                  </Heading>
+                  <div className="AccountCreator__subheading">
+                    {t("This will be used to unlock your wallet")}
+                  </div>
+                  <Form>
+                    <OnboardingOneCol>
+                      <div className="AccountCreator__form-rows">
+                        <Field name="password">
+                          {({ field }: FieldProps) => (
+                            <Input
```

Comment:
I'm noticing a style inconsistency in the active state for the field. It seems like if you tab from the second password field to the checkbox, the checkbox does not show the "active" border even though it is the currently active element. You can see in the video that once I tab back, it seems to trigger that style on subsequent tab navs to the field. It seems like maybe this style does not get set properly when the form is not in a dirty state?

https://github.com/user-attachments/assets/9dfbf2ca-1e24-45e2-9880-50605541720f

Reply:
Good callout! I'll take a look at this

Reply:
Hm, I do seem to see an active state when I tab to the checkbox. It's pretty light, but there is a difference. We could explore making this difference more apparent in SDS

Inactive:
<img width="62" alt="Screenshot 2024-11-26 at 3 22 03 PM" src="https://github.com/user-attachments/assets/e993bf09-0e8e-4584-8e73-47b1f11c0aa8">


Active:
<img width="93" alt="Screenshot 2024-11-26 at 3 21 54 PM" src="https://github.com/user-attachments/assets/a8598b8b-ccd3-483a-9b3c-2031af57794b">

Reply:
Hmm yeah I see the shadow now.  If you tab back after the first time you get the red active state border on the checkbox but for some reason not on the first time. I don't see any css for this though so it's an artifact of the sds styles for the checkbox.

Reply:
Oh the red outline is to signify that it's a required field and you haven't filled it out yet. That styles comes from Formik. So in this instance, that interaction makes sense as you would only want to alert the user that they missed a required field after the first time they tabbed away from the field